docs/admin/system-information.rst

```diff
@@ -1574,49 +1574,161 @@ Tables need to be recreated
    cluster check is failing. Follow the instructions below to get this cluster
    check passing.
 
-This check warns you if there are tables that need to be recreated for
-compatibility with future major versions of CrateDB.
+This check warns you if your cluster contains tables that you need to reindex
+before you can upgrade to a future major version of CrateDB.
 
-If you try to upgrade to the next major version of CrateDB with tables that
-have not been recreated, CrateDB will refuse to start.
+If you try to upgrade to a later major CrateDB version without reindexing the
+tables, CrateDB will refuse to start.
 
-To recreate a table, you have to create new tables, copy over the data and
-rename or remove the old table.
+CrateDB Table Version Compatibility Scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1) Use :ref:`ref-show-create-table` to get the schema required to create an
-empty copy of the table to recreate::
+CrateDB maintains backward compatibility for tables created in ``majorVersion - 1``:
 
-    SHOW CREATE TABLE your_table;
-
-2) Create a new temporary table, using the schema returned from
-:ref:`ref-show-create-table`::
-
-    CREATE TABLE tmp_your_table (...);
-
-3) Prevent inserts to the original table::
-
-    ALTER TABLE your_table SET ("blocks.read_only" = true);
-
-4) Copy the data::
+.. list-table::
 
-    INSERT INTO tmp_your_table (...) (SELECT ... FROM your_table);
+    * - Table Origin
+      - Current Version
+      - Current Version
+      - Current Version
+    * - 
+      - 3.x
+      - 4.x
+      - 5.x
+    * - 3.x
+      - ✔️
+      - ✔️
+      - ❌
+    * - 4.x
+      - ❌
+      - ✔️
+      - ✔️
+    * - 5.x
+      - ❌
+      - ❌
+      - ✔️
+
+
+Avoiding Reindex Using Partitioned Tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reindexing tables is an expensive operation which can take a long time. If you
+are storing time series data for a certain retention period and delete old
+data, it is possible to use the :ref:`partitioned_tables` feature to avoid
+reindex operations.
+
+You will have to partition a table by a column that denotes time. For example,
+if you have a retention period of 9 months, you could partition a table by a
```

Comment:
for numbers 0-9, it's good style to spell them out. so, "nine", in this instance

note: this doesn't apply to literals